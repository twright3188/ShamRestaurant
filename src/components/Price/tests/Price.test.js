import React from 'react';
import renderer from 'react-test-renderer';
import { ActivePrice, OldPrice } from '../index';

describe('price items render', () => {
    it('active price render', () => {
        const tree = renderer.create(<ActivePrice price='$40' />).toJSON();
        expect(tree).toMatchSnapshot();
    });
    it ('old price render', () => {
        const tree = renderer.create(<OldPrice price='$50' />).toJSON();
        expect(tree).toMatchSnapshot();
    });
});

