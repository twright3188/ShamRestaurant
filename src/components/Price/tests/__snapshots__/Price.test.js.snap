// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`price items render active price render 1`] = `
<Text
  style={
    Array [
      Object {
        "color": "#399AE5",
        "fontFamily": "Helvetica Neue",
        "fontWeight": "bold",
        "marginRight": 22,
      },
      Object {
        "fontSize": 35,
        "lineHeight": 39.97,
      },
    ]
  }
>
  $40
</Text>
`;

exports[`price items render old price render 1`] = `
<Text
  style={
    Array [
      Object {
        "color": "#C8CCC9",
        "fontFamily": "Helvetica Neue",
        "marginBottom": 4,
        "textDecorationLine": "line-through",
      },
      Object {
        "fontSize": 20,
        "lineHeight": 24,
      },
    ]
  }
>
  $50
</Text>
`;
